index.html

```diff
@@ -16,6 +16,11 @@
   <body>
     <header>
       <a href="#">My Logo</a>
+      <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li class="last">Contact</li>
+      </ul>
```

Comment:
The nav is the element of your HTML page that redirects you to other pages/sections of your website. Taking this into account, please add `a` tags to your `li` elements, directing them to the corresponding sections. This will make user navigation easier. 😉